geomesa-core/src/main/scala/org/locationtech/geomesa/core/iterators/IteratorFunctions.scala

```diff
@@ -0,0 +1,260 @@
+/*
```

Comment:
Perhaps add a Readme.md in geomesa-core that describes how the iterator stack is built up?